src/components/app/app.scss

```diff
@@ -1,28 +1,48 @@
 @import '../../styles/variables';
 
+//-- Theme colours as CSS Custom Properties --//
+
+.kui-theme--light {
+  --color-default: #{$color-light};
+  --color-default-alt: #{$color-dark};
+  --color-text: #{$color-text-light};
+  --color-text-alt: #{$color-text-light-alt};
+  --color-bg-1: #{$color-bg-light-1};
+  --color-bg-2: #{$color-bg-light-2};
+  --color-bg-3: #{$color-bg-light-3};
+  --color-bg-4: #{$color-bg-light-4};
+  --color-bg-alt: #{$color-bg-light-alt};
+  --color-sidebar-background: #{$color-bg-light-3};
+  --color-sidebar-border: rgba(0, 0, 0, 0.05);
```

Comment:
any reason this color is declared here instead of as a variable like the other colors above?

Reply:
Basically just because it wasn't necessary for anything - I want to move towards having just CSS custom properties and not Sass variables eventually, but I'm using both for now where they already existed. This one just wasn't a Sass variable previously so it didn't seem necessary to introduce one!

Reply:
I see, sounds good!